### PR TITLE
slimming down pantry

### PR DESCRIPTION
**Pull request by ttornese**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jumper_cables",
+  "name": "pantry",
   "description": "a starter kit for cool apps - built using https://github.com/este/este",
   "repository": "",
```
Comment (jks8787) [inline app.json]: should update that description too i think

File: app.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "jumper_cables",
+  "name": "pantry",
   "description": "a starter kit for cool apps - built using https://github.com/este/este",
```
Comment (jks8787) [inline app.json]: should update description too - i think

**Comment by jks8787** [comment]
look pretty good - but i think you will have to also update the readme

File: package.json [hunk 3]
```diff
@@ -13,10 +13,9 @@
     "test": "gulp test"
   },
   "engines": {
-    "node": "5.0.0"
+    "node": "6.3.0"
   },
   "dependencies": {
-    "@atk/pantry": "^1.0.0",
```
Comment (jaslakson) [inline package.json:19]: haha- nice...a recursive dependency!

File: app.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
   "repository": "",
   "logo": ",
```
Comment (somecallmejosh) [inline app.json]: Is this intended to be an empty string?

Comment (ttornese) [inline app.json]: that shouldn't be there. ill remove it and update my pull

**Comment by jaslakson** [comment]
👍 I say merge it and we can start working on the components. While doing so we can continue the cleanup effort.

**Pull request merged**